src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -18,8 +28,24 @@
      *                 - MONTH - название текущего месяца;
      *                 - DAY - текущий день (число месяца);
      **/
+
+    private static final String UKR_ZONE = "+02:00";
+    private static final DateTimeFormatter d_MMM_yyyy = DateTimeFormatter.ofPattern("dd MMM yyyy");
+    private static final DateTimeFormatter dd_MMMM_yyyy_HH_mm =
```

Comment:
better rename to CUSTOM_FORMATTER_WITH_TIME

Reply:
OK